docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+The user needs to be especially aware of this when specifying the maximum DN size for which the optimizer routine needs to size the pipe diameters. More information in subsection :ref:`subsec_V_Pipe_basic_attributes`.
+
+.. _subsec_V_Fluid_properties:
+
+Fluid properties
+^^^^^^^^^^^^^^^^
+
+* Constant density :math:`\rho` = 988 kg/m\ :sup:`3` (thus regardless the temperature). This density corresponds to a water temperature of about 50 :math:`^\circ C`.
+* Constant specific capacity :math:`c_p` = 4200 J/kg K (thus regardless the temperature). This specific heat corresponds to a water temperature of about 85 :math:`^\circ C`.
+
+.. _subsec_V_Ground_properties:
+
+Ground properties
+^^^^^^^^^^^^^^^^^
+
+The conductivity for the ground or soil is influenced a lot by all different factors depending on the type of soil and
+for example moist level. The NEN norm specifies that usually a value between 1.5 - 2 W/mK is used for wet soil and about 1.0 W/mK for dry soil.
+By default, the optimizer routine assume a value of 2.3 W/mK for the conductivity of the soil.
+Moreover, it assumes that the temperature of the soil, :math:`T_g` = 10.0 :math:`^{\circ} C`.
+
+.. _subsec_V_Pipe_conf_subsoil:
+
+Pipe configuration in subsoil
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The optimizer routine assumes that the pipes always come in pairs: one supply pipe and one return pipe that are placed parallel to each other in the subsoil.
+It also assumes that the supply and return pipes have the exact same properties.
+As the user, in the current version, cannot specify a distance between the two parallel pipe, it assumes that the distance from center to center,
+:math:`C`, twice the outer pipe diameter, :math:`D_{outer}` is, i.e:
+
+.. math::
+    :label: distance_pipes_optimizer
+
+    {C} = 2 D_{outer}
+
+With other words, it assumes that the is a distance of one pipe outer diameter between the supply and the return pipe in a pipe pair.
+
+Furthermore, the optimizer assumes that the pipes are buried and the depth of burial to the top of pipe, :math:`z^{\prime}`, is 1.0 m.
+Users cannot specify the depth of burial to the top of pipe themselves in the current version.
+The depth of burial to the center of pipe, :math:`z`, is computed by
+
+.. math::
+    :label: depth_pipes_optimizer
+
+    {z} = z^{\prime} + 0.5 D_{outer}
+
+.. _subsec_V_Heat_losses:
+
+Heat losses
+^^^^^^^^^^^
+
+The optimizer considers three types of heat losses in the pipelines,
+in accordance to the NEN-EN 13941+A1 :cite:p:`NEN-EN13941+A1`. 
+These are the heat losses through:
+
+* The pipe wall
+* Subsoil
+* Neighboring pipelines
+
+The optimizer routine does not account for losses at the surface (interface subsoil and the atmosphere).
+However, for completeness, we mention here that it does assume a heat transfer coefficient at surface of 15.4 W/m\ :sup:`2`/K
```

Comment:
The coefficient should have an effect, but since the losses to the surface are not modelled, it does not. So only for completeness, because it is hardcoded in the routine and I use it later for the validation with WANDA (so it is not a random value). In added "for completeness event-though it does not affect any results"